include/sisl/grpc/generic_service.hpp

```diff
@@ -20,7 +20,7 @@
 namespace sisl {
 
 using generic_rpc_handler_cb_t = std::function< bool(boost::intrusive_ptr< GenericRpcData >&) >;
```

Comment:
This handler_cb_t seems like a redefinition from `rpc_server.hpp`; is it needed here?

Reply:
common makes more sense, I will move it there